uploader/setup.sh

```diff
@@ -18,11 +18,16 @@ curl -o ${UPLOADER_HOME}/pantheon.py https://raw.githubusercontent.com/redhatacc
 echo "Ensure uploader script is executble"
 chmod +x ${UPLOADER_HOME}/pantheon.py 
 
-# Create a symlink to /usr/local/bin if not exists
+# Create a symlink to ~/bin/pantheon if not exists
 if [ ! -h "$PANTHEON_BIN" ]; then
   echo "Create a symlink to $PANTHEON_BIN"
-  sudo ln -s ${UPLOADER_HOME}/pantheon.py $PANTHEON_BIN
+  ln -s ${UPLOADER_HOME}/pantheon.py $PANTHEON_BIN
 fi
+# Add a new directory into PATH variable
+echo "Adding ${PANTHEON_BIN} to PATH variable"
+PATH=$PATH:$PANTHEON_BIN
+export PATH
+echo $PATH | grep $PANTHEON_BIN
```

Comment:
Can you please delete these 5 lines? Here's my thought:

- I'm 90% sure that an `export` all by itself will only last for that terminal session. It won't affect any future terminals nor persist through a system restart, so this does nothing permanent.
- `~/bin` is already in the default path, at least on my machine:
>$ echo $PATH
>/usr/local/bin:/usr/bin:/bin:/usr/local/sbin:/usr/sbin:/home/ben/.local/bin:/home/ben/bin

So I don't believe that these lines do anything meaningful.

Reply:
good point. removed!